### PR TITLE
feat(webserver): Include ServiceName on Common Responses

### LINKED ISSUE #1088: Include ServiceName on common responses
As of go-mod-core-contracts `v2.2.0-dev.9` ServiceName is available on the following responses:

- ping
- version
- metrics
- config

Will need to update SDK and use the new constructors where needed

### PR DESCRIPTION
**Pull request by AlexCuse**
Included on ping, version, metrics, config responses.

Fixes #1088

Signed-off-by: Alex Ullrich <alex.ullrich@gmail.com>

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-sdk-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [x] I am not introducing a new dependency (add notes below if you are)
     - go.mod diff from contract changes
- [x] I have added unit tests for the new feature or bug fix (if not, why?)
- [ ] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
    unit tests sufficient
- [ ] I have opened a PR for the related docs change (if not, why?)
    not sure needed

### CODE REVIEW

File: example/cmd/device-simple/Attribution.txt
Comment (cloudxxx8) [inline example/cmd/device-simple/Attribution.txt:163]: could you please also remove the section of 
```
go-kit/kit (MIT) https://github.com/go-kit/kit
https://github.com/go-kit/kit/blob/master/LICENSE
```

**Review by cloudxxx8 — CHANGES_REQUESTED** [review]
the go-mod-core-contract includes the breaking change, we should not merge this PR until it is resolved

File: internal/controller/http/common.go
Comment (cloudxxx8) [inline internal/controller/http/common.go:27]: this kind of signature change will break all the original code after upgrade go-mod-core-contracts

Comment (lenny-intel) [reply internal/controller/http/common.go:27]: @cloudxxx8 , this is using the upgraded go-mod-core-contracts and adjusting for the signature change we agreed was ok since it is only use by our internal code.

File: example/cmd/device-simple/Attribution.txt
Comment (lenny-intel) [reply example/cmd/device-simple/Attribution.txt:163]: @AlexCuse,  so it looks like `go-kit/log` replaced use of `go-kit/`

**Review by lenny-intel — APPROVED** [review]
LGTM, once address @cloudxxx8 's comment on attribution file

**Comment by lenny-intel** [comment]
@AlexCuse , need to also update the openApi docs (swagger) to show the addition of service name in the response.

File: internal/controller/http/common.go
Comment (cloudxxx8) [reply internal/controller/http/common.go:27]: @lenny-intel go-mod-core-contracts is a public module, but if you have discussed and agreed, I am fine with this change.

Comment (lenny-intel) [reply internal/controller/http/common.go:27]: @cloudxxx8 , you agreed here: https://github.com/edgexfoundry/go-mod-core-contracts/pull/695#discussion_r791833007 
:-)

**Pull request closed without merging**
